src/middleware.js

```diff
@@ -41,19 +39,19 @@ class KoaJoiValidatorMiddleware {
     }
   }
 
-  _handleValidationError(error) {
+  _handleValidationError(ctx, next, error) {
     if (error instanceof Joi.ValidationError === false) {
```

Comment:
We can either remove this line (since general errors will not be caught any ways), or add another optional input parameter for a general error handler that is called when set.

Reply:
✅